lib/perl/Genome/Model/Tools/FastTier/TierBitmasks.pm

```diff
@@ -116,174 +61,67 @@ sub create {
+    my $tier2 = $self->shadow_genome(\%genome);
+    my $repeatmasker_regions = $self->shadow_genome(\%genome);
 
     #now do regulatory regions
-    #
     #First determine repeats
-    my $repeatmasker_regions = $self->shadow_genome(\%genome);
-
-    my @repeatmasker_files = glob($self->ucsc_directory."/rmsk/*");
-    for my $file (@repeatmasker_files) {
-        my $fh = Genome::Sys->open_file_for_reading($file);
-        while(my $line = $fh->getline) {
-            chomp $line;
-            my @fields = split /\s+/, $line;
-            my ($chr, $start, $end) = @fields[5,6,7];
-            $chr =~ s/chr//g;
-            $self->add_range_to_set($repeatmasker_regions,$chr, $start, $end); 
-        }
-        $fh->close;
-    }
-    print STDERR "Calculated repeatmasker regions\n";
-    $self->debug_message("Calculated repeatmasker regions\n");
-
-    #now take union and calculate the coverage
-
-
-    #next build up the regulatory annotated regions
-    my $regulatory_regions = $self->shadow_genome(\%genome);
-    #my @files = qw| targetScanS.txt oreganno.txt tfbsConsSites.txt vistaEnhancers.txt eponine.txt firstEF.txt wgEncodeUcsdNgTaf1ValidH3K4me.txt wgEncodeUcsdNgTaf1ValidH3ac.txt wgEncodeUcsdNgTaf1ValidRnap.txt wgEncodeUcsdNgTaf1ValidTaf.txt polyaDb.txt polyaPredict.txt switchDbTss.txt encodeUViennaRnaz.txt laminB1.txt |;
-    my @regulatory_regions = glob($self->ucsc_directory."/regulatory_regions/*");
-
-    for my $file (@regulatory_regions) {
-        my $fh = Genome::Sys->open_file_for_reading($file);
-        while(my $line = $fh->getline) {
-            chomp $line;
-            my ($bin, $chr, $start, $end, ) = split /\t/, $line; #ignoring scores on these tables (may be bad)
-            $chr =~ s/chr//g;
-            $self->add_range_to_set($regulatory_regions,$chr, $start, $end); 
-        }
-        $fh->close;
-    }
-    print STDERR "Calculated Tier2 regulatory regions\n";
-    $self->debug_message("Calculated Tier2 regulatory regions\n");
-
-    #no bins in this file
-    my @cpg_islands = glob($self->ucsc_directory."/cpg_islands/*");
-    for my $cpg_islands (@cpg_islands){
-        my $fh = Genome::Sys->open_file_for_reading($cpg_islands);
-        while(my $line = $fh->getline) {
-            chomp $line;
-            my ($bin, $chr, $start, $end, ) = split /\t/, $line; #ignoring scores on these tables (may be bad)
-            $chr =~ s/chr//g;
-            $self->add_range_to_set($regulatory_regions,$chr, $start, $end); 
-        }
-        $fh->close;
+    if ($self->ucsc_directory) {
+        #Tier2 contains all coding bases (silent mutations), as well as conserved bases via phastConsElements and non-repeat regulatory
+        #Scan through phastConsElements
+        $tier2 = $self->create_conserved_bitmask(\%genome);
+        print STDERR "Calculated Tier2 conserved set\n";
+        $self->debug_message("Calculated Tier2 conserved set\n");
```

Comment:
It's not something new in this PR (in fact it's from 2012!), but this `print STDERR` followed by `debug_message` pattern is strange.